### PR TITLE
quayio: Redirect user to billing page when starting free trial (PROJAUAY-4130)

### PR DESCRIPTION
**Pull request by jonathankingfc**
- When a user clicks 'Start Free Trial', they should be redirected to the user billing page after signing in with RHSSO

### CODE REVIEW

File: static/js/services/plan-service.js [hunk 1]
```diff
@@ -69,7 +69,8 @@ function(KeyService, UserService, CookieService, ApiService, Features, Config, $
         if (planService.isOrgCompatible(plan)) {
           $location.path('/organizations/new').search('plan', planId);
         } else {
-          $location.path('/user').search('plan', planId);
+          const username = UserService.currentUser().username;
+          $location.path(`/user/${username}/billing`);
```
Comment (Sunandadadi) [inline static/js/services/plan-service.js:73]: Just want to confirm if this the correct endpoint as I was not able to find a flask route for it.

Comment (jonathankingfc) [reply static/js/services/plan-service.js:73]: This isn't an API route. It is a frontend route. You can check the frontend route here `https://quay.io/user/{your_username}/billing`

Comment (Sunandadadi) [reply static/js/services/plan-service.js:73]: Oh! Got it. Thanks!

**Review by Sunandadadi — APPROVED** [review]

**Comment by Sunandadadi** [comment]
lgtm

**Pull request merged**